lib/ios/RNNModalManager.m

```diff
@@ -105,6 +109,10 @@ - (void)dismissedModal:(UIViewController *)viewController {
 	[_delegate dismissedModal:viewController];
 }
 
+- (void)presentationControllerDidDismiss:(UIPresentationController *)presentationController {
+	[_presentedModals removeObject:presentationController.presentedViewController];
```

Comment:
We are missing here:
```
[_delegate dismissedModal:presentationController.presentedViewController];
```

Without it, `events().registerModalDismissedListener` is not triggered correctly.